chalk-solve/src/infer.rs

```diff
@@ -204,6 +204,36 @@ impl<I: Interner> InferenceTable<I> {
             InferenceValue::Bound(_) => panic!("var_universe invoked on bound variable"),
         }
     }
+
+    /// Check whether the given substitution is the identity substitution in this
```

Comment:
I feel like we .. have code for this? But maybe not in the inference context

Reply:
I don't know, though this function did exist before and got removed after the recursive solver got removed